app/Http/Controllers/Api/Client/Store/ServerController.php

```diff
@@ -87,6 +92,6 @@ public function store(CreateServerRequest $request): JsonResponse
             'store_databases' => $user->store_databases - $request->input('databases'),
         ]);
 
-        return new JsonResponse([], Response::HTTP_NO_CONTENT);
+        return new JsonResponse(['success' => true, 'data' => ['id' => $server->uuidShort]]);
```

Comment:
Why not make this just `success => true, id => serverUUID`? Then you won't need to do `data.data.id` at the frontend.